plugins/module_utils/vmware.py

```diff
@@ -946,6 +946,13 @@ def quote_obj_name(object_name=None):
     return object_name
 
 
+def dvs_supports_mac_learning(dvs):
```

Comment:
Do you think that this function implements into Pyvmomi class is better than outside the class?    
What do you think? @Akasurde @mariolenz

Reply:
@sky-joker @Akasurde Honestly, I'm not sure. My reasoning was that most modules (which inherit from PyVmomi) don't care if a switch supports MAC learning or not, so the class shouldn't provide a function to check this.

But I'm ~~a doctor~~ an admin and not a developer, so maybe there are good reasons to make this function a method of the class I just don't know about.

Reply:
ok.
I thought about that again, I think it hasn't a problem even the function is defined outside of PyVmomi class.